test/arbitrable-kitty.js

```diff
@@ -732,7 +732,7 @@ contract('ArbitrableKitty', accounts => {
         "should be under A's custody"
       )
 
-      await increaseTime(60 * 60 * 24 * 7)
+      await increaseTime(60 * 60 * 24 * 7 + 1 )
```

Comment:
Yeah, because time outs are `>` and not `>=`, nice catch.

Reply:
Hopefully this fixes it. I will run rebuild a couple of times before removing the [WIP] prefix though